packages/ui-contract-editor/src/lib/components/actions.js

```diff
@@ -17,3 +17,5 @@ export const headerGenerator = (templateTitle, inputTitle) => {
   const truncatedTitle = ((header.length > 54) ? (`${header.slice(0, 54)}...`) : header);
   return truncatedTitle;
 };
+
+export const childReducer = (children) => children.reduce((acc, cur) => `${acc}${cur.text}`, '');
```

Comment:
Why not render as Slate DOM?

Reply:
Maybe that's what it does. Just unclear what the purpose of the childReducer is.

Reply:
It links all the `text` values through the children array so we can have a plain text version to put in the tooltips.